packages/extension-api/src/extension-api.d.ts

```diff
@@ -1103,6 +1103,10 @@ declare module '@podman-desktop/api' {
      */
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     commandArgs?: any[];
+    /**
+     * Marks an item to have a badge.
+     */
+    badged: boolean;
```

Comment:
Should be optional otherwise looks like an API breakage ?

Reply:
definitely, I will fix that later